codeship-services.yml

```diff
@@ -35,8 +35,9 @@ e2e:
 
 # 'db' service provides a postgres db with postgis
 db:
-  image: mdillon/postgis:10-alpine
+  image: postgis/postgis:13-3.2-alpine
```

Comment:
Switched to using [postgis image](https://registry.hub.docker.com/r/postgis/postgis/) over [mdillion image](https://hub.docker.com/r/mdillon/postgis/) as it looks like mdillion is no longer being maintained (last update was 3 years ago and has no postgres13 support).